src/config/Config.js

```diff
@@ -119,6 +119,8 @@ class Config {
   toEnvs() {
     return {
       COMPOSE_PROJECT_NAME: `dash_masternode_${this.getName()}`,
+      MN_HOME_DIR: '~/.mn', // this.container.resolve('homeDirPath'),  How do I get this variable?
```

Comment:
Maybe we should export it here? https://github.com/dashevo/mn-bootstrap/blob/6dd9a0b960fa7361bc0e4f03c6758a7082e32011/src/docker/DockerCompose.js#L266
And please don't forget about `config:envs` command.

Reply:
How and why do I export it there, what process did you use to find out this is the place where it needs to be exported? When is `getOptions(envs)` function called and how does it relate to `config:envs` and `Config.toEnvs()`?? How can I visualise these relationships between functions?

Reply:
Just pass home as a dependency to DocketCompose file so it will be available in getOptions. This method is called before envs passed to docker-compose cli. So it reasonable to enrich config envs with this system env in this place.

Reply:
OK, I did what you said and it works, but I don't understand how. Also now nothing has changed in `Config.js`, and the variable doesn't appear in the output of `config:envs`. I still don't know what result you want to achieve with this, without knowing why we are doing it, I don't know what to do next.

Reply:
I'm sorry, I'm not so good at writing. Let me try one more time ). We do not have this homedir option in the config and it's kind of out of its scope logically. So, it makes sense just to add it somewhere on a lower level where we passing env variables to docker-compose CLI for example. There are two places where we are doing this:
- `DockerCompose#getOption` (bad naming for sure, must be something like `createOptions`) 
- `config:env` command

So we can enrich envs with this system one in these two places and logically it will be ok.

Reply:
Hmm, maybe I was getting `DockerCompose#getOptions` confused with `Config#getOptions` while trying to use the stack trace in the debugger... I added it to `envs.js` instead of `Config.js` now, so it is added in the command and not in the class. Still not clear why we need to export this, since it is a constant and not a variable.

Reply:
I remember we discussed making it an environment variable to allow people to specify the path. It makes sense for the deployment tool for example. We didn't do that yet?

Reply:
No, I couldn't figure out how to pass command line flags to awilix. I thought we agreed to do it in a separate PR.

Reply:
Ah I see. Actually you don't need to pass command line flags. All env variables in `proccess.env` so we usually pass  it to `createDIContainer` as an options param https://github.com/dashevo/js-drive/blob/v0.17-dev/scripts/abci.js#L10

Reply:
I think I see, so `process.env` contains the current user environment and we can just check if a var is set there? I defined it to check if `MN_HOME_DIR` is set, is that right?

Reply:
Yeah, something like:

```js
const homeDirPath = process.env.MN_HOME_DIR || path.resolve(os.homedir(), '.mn');
```